src/roswire/ros2/state.py

```diff
@@ -10,6 +10,13 @@
 
 from ..common import SystemState
 
+ACTION_CLIENTS = "act_cli"
```

Comment:
Minor nitpick: It might be worth adding a leading underscore to these variables to avoid them being included in the module's public API (i.e., `_ACTION_CLIENTS`).